server/routerlicious/packages/services-utils/src/auth.ts

```diff
@@ -154,6 +154,9 @@ export function verifyStorageToken(
         try {
             await tenantManager.verifyToken(claims.tenantId, token);
         } catch (error) {
+            if (error instanceof NetworkError) {
+                return res.status(error.code).send(error.message);
```

Comment:
Why not send the whole error object as JSON?  the canRetry and retryAfter stuff would be useful to the caller, I assume?

Reply:
The driver does not parse the response body for canRetry (relies on error code for that). As for `retryAfter` you are totally correct. If `retryAfter` (and/or retryAfterMs) exists on the NetworkError we should be returning an object with properties message and retryAfter(+retryAfterMs).